config/Make.project.rules

```diff
@@ -665,7 +665,7 @@ $2_distclean::
 #
 ifeq ($$($2_devinstall),yes)
 ifneq ($$(and $$($2_components),$(filter library,$3),$$(filter $(includedir)/%,$$($2_includedir))),)
-$$(eval $$(call install-data-files,$$(wildcard $$($2_includedir)/*.h),$(includedir),$(install_includedir),$2_install))
+$$(eval $$(call install-data-files,$$(shell find $$($2_includedir) -iname '*.h'),$(includedir),$(install_includedir),$2_install))
```

Comment:
It requires to use `shell find` but using the makefile wildcard was causing troubles relative to rules getting override.